generalized_pose_msgs/GeneralizedPose generalized_pose
float64 time